npc/custom/quests/valhallen.txt

```diff
@@ -43,26 +48,71 @@ prontera,147,171,5	script	Lenneth	811,{
 		case 5:
 			mes "[Lenneth]";
 			mes "The ^3355FFMegingjard^000000 is the powerful Belt of Thor.";
-			setarray .@Items[0],2629,7058,1,2627,1,969,10,726,10,984,5;
-			break; }
+			setarray .@Items[0],2629,7058,1,2627,1,969,20,726,10,984,10;
+			break;
+		case 6:
+			mes "[Lenneth]";
+			mes "The ^3355FFAsprika^000000 is the light of gods.";
+			setarray .@Items[0],2541,7835,1,7836,1,7837,1,7838,1,2513,1,7063,100,7291,10,7293,10,985,20;
+			break;
+		case 7:
+			mes "[Lenneth]";
+			mes "The ^3355FFBrynhild^000000 is the warrior's armor.";
+			setarray .@Items[0],2383,7830,1,7831,1,7832,1,7833,1,7834,1,7510,100,2357,1,969,10,985,20;
+			break;
+		case 8:
+			mes "[Lenneth]";
+			mes "The ^3355FFHervor^000000 is named after the valkyrie who ensures victory for her warriors.";
+			setarray .@Items[0],2181,6595,2,6596,1,6594,4,6597,3,6602,1,6604,1,6605,1,2115,1;
```

Comment:
May you add Muspellium and Essence_Of_Rune menu creation (like Gleipnir to make Megingjard) ?
https://github.com/rathena/rathena/blob/woe_te-restrictions/npc/re/quests/woe_te/te_goditem_alde1.txt#L329
https://github.com/rathena/rathena/blob/woe_te-restrictions/npc/re/quests/woe_te/te_goditem_alde1.txt#L349
http://irowiki.org/wiki/God_Items_Quest_3#cite_ref-1